index.html

```diff
@@ -0,0 +1,177 @@
+        <p class="p-4 border border-3">
+          The <strong>World Snooker Championship 2022</strong> was organised by the World Snooker Tour
+          and sponsored by sports betting company Betfred. It featured a total prize fund of £2,395,000,
+          of which the winner received £500,000. The main stage of the tournament featured 32 players: the top 16
+          players from the snooker world rankings and an additional 16 players from the qualifying rounds.
+        </p>
+        <h2 class="event-date">2022.04.16(Sat) ~ 2022.05.02(Mon)</h2>
+        <p class="event-location">
+          @ Crucible Theatre in Sheffield, England
+        </p>
+      </section>
+    </article>
+    <article class="program-container d-flex flex-column align-items-center p-3" id="program-section">
+      <h2 class="section-heading">Main Program</h2>
+      <hr />
+      <section class="row gap-3 p-3 justify-content-center">
+        <section class="program-card col-md-2">
+          <div class="icon-container">
+            <i class="fa-solid fa-chalkboard-user"></i>
+          </div>
+          <h3 class="program-title">Lecture</h3>
+          <p class="p-1">
+            Listen to the snooker lessons taught by the most-experienced
+            international grandmasters.
+          </p>
+        </section>
+        <section class="program-card col-md-2">
+          <div class="icon-container">
+            <i class="fas fa-chess-board"></i>
+          </div>
+          <h3 class="program-title">Match</h3>
+          <p class="p-1">
+            Watch the exciting snooker matches by the top players and learn from
+            them.
+          </p>
+        </section>
+        <section class="program-card col-md-2">
+          <div class="icon-container">
+            <i class="fas fa-chess-queen"></i>
+          </div>
+          <h3 class="program-title">Practice</h3>
+          <p class="p-1">
+            Play snooker with other players over a chessboard, rather than
+            playing online.
+          </p>
+        </section>
+        <section class="program-card col-md-2">
+          <div class="icon-container">
+            <i class="fas fa-book"></i>
+          </div>
+          <h3 class="program-title">Theory</h3>
+          <p class="p-1">
+            Study the important theory about the openings, middle game and
+            endgame.
+          </p>
+        </section>
+        <section class="program-card col-md-2">
+          <div class="icon-container">
+            <i class="fas fa-users"></i>
+          </div>
+          <h3 class="program-title">Connect</h3>
+          <p class="p-1">
+            Become part of the amazing snooker community to discuss, study and
+            play.
+          </p>
+        </section>
+      </section>
+      <button type="button" id="join-btn">Join Event</button>
+    </article>
+    <article class="feature-container d-flex flex-column align-items-center p-3 bg-white" id="player-section">
```

Comment:
- [ ] The speaker section should lso be wrapped in a section tag and not an `article` tag for semantic reasons.